app/views/application/_flash.html.erb

```diff
@@ -0,0 +1,8 @@
+<div class="container">
+  <% flash.each do |type, msg| %>
+    <div class="alert <%= bootstrap_class_for_flash(type) %> alert-dismissable fade show">
+      <%= msg %>
+      <button class="close" data-dismiss="alert">x</button>
+    </div>
+  <% end %>
+</div>
```

Comment:
css/js magic from the interwebs!

Reply:
Hermione would be proud!

Reply:
Ooh `alert-dismissable fade show` sounds like it does something cool.=